integration_test/third_party_apps_data/applications/zookeeper/centos_rhel/install

```diff
@@ -4,7 +4,7 @@ sudo yum install -y curl java
 
 sudo mkdir -p /opt/zookeeper/stage
 sudo curl \
-    "https://dlcdn.apache.org/zookeeper/zookeeper-3.7.0/apache-zookeeper-3.7.0-bin.tar.gz" \
+    "https://dlcdn.apache.org/zookeeper/zookeeper-3.7.1/apache-zookeeper-3.7.1-bin.tar.gz" \
```

Comment:
[Optional] If we change this to `https://archive.apache.org/dist/zookeeper/zookeeper-3.7.1/apache-zookeeper-3.7.1-bin.tar.gz`, it won't disappear and break our tests. On the other hand, we won't know to update it either. Tradeoffs…

Reply:
I'll do this in a follow up PR. thanks for the suggestion.